pkg/bytecode/value.go

```diff
@@ -59,3 +60,38 @@ func (s stringVal) Equals(v value) bool {
 	}
 	return s == s2
 }
+
+type arrayVal struct {
+	Elements []value
+}
+
+func (a arrayVal) Type() *parser.Type {
+	// Revisit this when adding the typeof builtin: https://github.com/evylang/evy/pull/305/files#r1531149977
+	return parser.GENERIC_ARRAY
+}
+
+func (a *arrayVal) String() string {
```

Comment:
Sorry, my previous comment on the pointer receiver applied to all the methods, and passing this around as a pointer to satisfy the interface, when a value type would work just fine too. I don't see the need for this to be used as a pointer at all.